webassembly/app.html

```diff
@@ -223,6 +221,27 @@ <h1 class="title">F3D Web</h1>
         // do a first render and start the interactor
         Module.engineInstance.getWindow().render();
         Module.engineInstance.getInteractor().start();
+
+        // Parse search-query mesh url-param or load default mesh file
+        const params = new URLSearchParams(window.location.search);
+        const mesh_url = params.get("mesh");
+        if (mesh_url) {
+          fetch(mesh_url)
+          .then((response) => {
+            if (!response.ok) {
+              throw new Error(`HTTP error, status = ${response.status}`);
+            }
+            return response.arrayBuffer();
+          })
+          .then((buffer) => {
+            // Convert buffer to Uint8Array and openFile
+            Module.FS.writeFile('mesh_urlparam', new Uint8Array(buffer));
+            openFile('mesh_urlparam');
```

Comment:
```suggestion
            const name = "..."; // retrieve name from mesh_url
            Module.FS.writeFile(name, new Uint8Array(buffer));
            openFile(name);
```
`f3d` needs the file extension to know the format and open the file

Reply:
Ah that's right, is the extension parsed from the filename written to Module.FS? 
In that case, the mesh url has to end with the extension, and filename be retrieved from it, correct? 
Otherwise, we can as well add another url-param, like `loader` or `extension`, which if present will enforce loading with the corresponding loader, and if not present, will default to parsing extension from url. Would that make sense ?

Reply:
> Ah that's right, is the extension parsed from the filename written to Module.FS?

yes this filename can be made up but it need the proper extension

> Otherwise, we can as well add another url-param, like `loader` or `extension`, which if present will enforce loading with the corresponding loader, and if not present, will default to parsing extension from url. Would that make sense ?

It does make sense, if the model was a glTF file but there was no way to tell from the url alone  (e.g. `http://example.com/foo/bar`) then you'd need some way to pass the format so that you could write the file as `bar.gltf` in the filesystem

Reply:
Alright thanks for the precision that the file extension should be passed as filename. 
This extension can be enforced by user via url-param first, simply `extension` which would be appended to a placeholder name. 
At some point, one might try to extract it from the `Response Headers` as advised [here](https://github.com/kovacsv/Online3DViewer/issues/432), a bit trickier to parse, like `Content-Disposition: attachment; filename=odm_textured_model_geo.obj; filename*=UTF-8''odm_textured_model_geo.obj`